examples/elasticsearch/tilenol.yml

```diff
@@ -10,10 +10,10 @@ layers:
     minzoom: 14
     source:
       elasticsearch:
-        host: localhost
+        host: elasticsearch
```

Comment:
Is it possible for docker-compose to use the host network instead of a virtual network? If so, it would allow this configuration file to work for vanilla Docker, docker-compose, and non-Docker based local ES instances.

Reply:
In fact, I'd personally prefer to be able to use `localhost` by default in this example, rather than relying on a virtual host.